patches/0001-dummy-patch-to-support-submodule-for-module-import.patch

```diff
@@ -0,0 +1 @@
+This is a dummy patch.
```

Comment:
```suggestion
This patch exists purely so that `patches` folder can be non-empty and the patches workflow can be invoked. See https://github.com/pulumi/pulumi-pagerduty/pull/546 for details.
```